src/main/viz/GenomeTrack.js

```diff
@@ -35,10 +35,11 @@ function renderGenome(ctx: DataCanvasRenderingContext2D,
   if (mode != DisplayMode.HIDDEN) {
     ctx.textAlign = 'center';
     if (mode == DisplayMode.LOOSE) {
-      ctx.font = style.LOOSE_TEXT_STYLE;
+      //ctx.font = style.LOOSE_TEXT_STYLE;
     } else if (mode == DisplayMode.TIGHT) {
-      ctx.font = style.TIGHT_TEXT_STYLE;
+      //ctx.font = style.TIGHT_TEXT_STYLE;
     }
+    ctx.font = String(Math.min(pxPerLetter, 12)) + "px" + ` 'Helvetica Neue', Helvetica, Arial, sans-serif`;
```

Comment:
pxPerLetter is for x axis size adjustment, not height adjustment. See https://github.com/hammerlab/pileup.js/blob/master/src/main/viz/GenomeTrack.js#L65, where they use pxPerLetter to size the rectangle widths.

This will not work for all cases (see screenshot below)

![screen shot 2019-03-06 at 10 41 44 am](https://user-images.githubusercontent.com/6503416/53905071-75ab9b80-3ffc-11e9-98a0-0e55f33a77ca.png)

You will need to re-adjust where the letters are printed : 

```
 ctx.textBaseline = "middle";
 ctx.fillText(letter, scale(1 + 0.5 + pos), adjustedHeight/2); 
```

where adjustedHeight is calculated from a scaling factor based on the display being used.